src/system.cpp

```diff
@@ -106,6 +106,35 @@ bool System::command_response(const char * value, const int8_t id, JsonObject &
     return true;
 }
 
+// output all the EMS devices and their values, plus the sensors and any custom things
+// basically dump everything we know about
+bool System::command_values(const char * value, const int8_t id, JsonObject & output) {
+    DynamicJsonDocument doc(EMSESP_JSON_SIZE_XXXLARGE);
+    JsonObject          device_output;
+
+    for (const auto & device_class : EMSFactory::device_handlers()) {
+        for (const auto & emsdevice : EMSESP::emsdevices) {
+            if (emsdevice->device_type() == device_class.first) {
+                std::string title = emsdevice->device_type_2_device_name_translated() + std::string(" ") + emsdevice->to_string();
+                device_output     = output.createNestedObject(title);
+                emsesp::EMSdevice::export_values(emsdevice->device_type(), device_output, id, EMSdevice::OUTPUT_TARGET::API_VERBOSE);
+            }
+        }
+    }
+
+    // Custom entities
+    device_output = output.createNestedObject("Custom Entities");
+    EMSESP::webCustomEntityService.get_value_info(device_output, "");
```

Comment:
add `webSchedulerService` here.

Reply:
I decided to leave the Scheduler out since it doesn't have any entity values - they are just records.

Maybe add a new system command that shows the scheduler details? That's useful since the current 'call scheduler info' only shows the name and whether its on/off, which isn't that useful.

Reply:
The scheduler have entity/command to activate/deactivate the schedule. This was for replace the `forceheatingoff`, sending `selflowtemp 0` every minute to stop the boiler, and disable scheduler to activate the boiler again.
Maybe it's a nice addition to also manipulate the schedule command value via command. But difficult to add in the command structure. 

I wonder if we can add custom RAM entities only stored in ems-esp and use them for setting a variable value. E.g. in custom use devcie/telegram type 0x0 for storing 256 bytes. Custom entity: `myseltemp, device 0, type 0, offset 0, UINT`, Scheduler: `every minute, command: boiler/seltemp, value: custom/myseltemp`. Then the flowtemp can be set via mqtt: custom/myseltemp and is used by the scheduler.

Reply:
I see, That's quite clever. How about instead of using custom entities to store variables we could create a new type called Variables. Maybe limit to 10 max, as strings with a default value that is stored in LittleFS and loaded at boot.

Reply:
moved to https://github.com/emsesp/EMS-ESP32/issues/1423